nibabel/parrec.py

```diff
@@ -244,6 +245,9 @@ class PARRECError(Exception):
     pass
 
 
+# Value after colon may be absent
+GEN_RE = re.compile(r".\s+(.*?)\s*:(.*)")
```

Comment:
Instead of `value.strip()` later, why not the following regex, which simply changes the `\s+` to a `\s*`?

    GEN_RE = re.compile(r".\s+(.*?)\s*:\s*(.*)")